src/terminal/terminal.c

```diff
@@ -504,6 +505,13 @@ guac_terminal* guac_terminal_create(guac_client* client,
     /* Configure backspace */
     term->backspace = options->backspace;
 
+    /* Configure the family of codes for function keys and the keypad */
+    if (options->func_keys_and_keypad == NULL || options->func_keys_and_keypad[0] == 0) {
+        term->func_keys_and_keypad = GUAC_TERMINAL_FUNC_KEYS_AND_KEYPAD_DEFAULT;
+    } else if (strcmp(options->func_keys_and_keypad, "vt100plus") == 0) {
```

Comment:
Please don't cuddle the `else if`. See: https://cwiki.apache.org/confluence/display/GUAC/Contribution+and+Style+Guidelines#ContributionandStyleGuidelines-Braces